src/guacamole-docker.md

```diff
@@ -782,6 +786,36 @@ SAML authentication extension.
   that may be helpful in debugging problems with SAML authentication. This
   is optional and defaults to false - debugging will not be enabled.
 
+(guacamole-docker-history-recording-storage)=
+
+### History Recording Storage Extension
+
+The extension that enables viewing historical recordings from within the
+Guacamole Client interface can be enabled by settings the search path
+variable, as noted below, to a location where the extension will look
+to find available recordings.
+
+When setting this up in a container environment, you'll likely need
+to use volumes to make the same directory available to both the
+guacd container and the guacamole (client) container. Setting up these
+volumes is beyond the scope of this document.
```

Comment:
Should we include the UIDs/GIDs used by the Guacamole and guacd images? Could be good to provide known-good values.

Reply:
Sure, happy to add those values in here.